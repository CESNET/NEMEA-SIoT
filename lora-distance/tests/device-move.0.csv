time TIMESTAMP,double RSSI,string PHY_PAYLOAD,uint64 DEV_ADDR,uint8 STATUS
2017-09-07T19:38:35.000,-119,400910012600840501E410666602,637603849,16
2017-09-06T19:32:47.000,-117,40D12B0126007B4501A5469B48533D6D97C2230252C0A6BBE873202033D703533961,637610961,16
2017-09-09T12:52:09.000,-120,402929012680100201D71D4BDBA20705119A304E1710782914,637610281,16
2017-09-08T04:03:22.000,-115,40D12B0126007C4501BD4CA9CE935FB2E6E1CE8D0C3BDD20AD4F0451096A25652FF7,637610961,16
2017-09-08T06:35:53.000,-120,40EC13012680F57701895399EEB6A0,637604844,16
2017-09-06T18:03:27.000,-119,4009100126008605014EDA100BA7,637603849,16
2018-03-20T18:46:41.000,-115,7C561EAF51621B4B710A2D42E4963CE3080A151FD2F567666FB7BEA5E41B255245841EE25CC05503A1C75BE038DC200E869F0B0E0C9CBF01586A0E,1370431062,16
2017-09-08T17:14:26.000,-116,40D12B0126007D4501D0391E4AC46092075E0F789EF3D19B130B23F3A591B9F38FB9,637610961,16
2017-09-08T17:14:26.000,-118,4009100126008705012018333C96,637603849,16
2017-09-09T10:34:47.000,-119,40D12B0126007E450138D08C539AD21CB50BDC8CCD0452EEF8E113BD01B95ABB6DC9,637610961,16
2017-09-08T10:35:58.000,-119,4009100126008805013F02960340,637603849,16
2017-09-06T23:43:54.000,-119,40D12B012600804501BAE2318377CC5CC9FC2B97EFD8AD7E9BA52D41DF43BF200747,637610961,16
2017-09-08T08:05:19.000,-118,4009100126008A05010A3DBF79AB,637603849,16
2017-09-08T11:53:44.000,-118,40EC13012680F67701523967E84489,637604844,16
2017-09-09T09:45:25.000,-117,40C910F1D056DB050170D788D162,3505459401,16
2017-09-06T18:39:29.000,-111,4009100126008C050168845FDEA7,637603849,16
2017-09-06T20:35:46.000,-117,4009100126008D0501A294E6AA16,637603849,16
2017-09-07T03:14:35.000,-113,40D12B01260083450181570DDD4BF729D18F780BBB3224F5FD1FAFF1AFADC6F0F548,637610961,16
2017-09-07T19:38:29.000,-118,4009100126008E0501B2B06A5444,637603849,16
2017-09-09T02:41:43.000,-120,40D12B012600844501949CC6332E8D20D507EE772BA6E02B26CB9E0CEBD81ADD2356,637610961,16
2017-09-06T15:11:49.000,-116,4009100126008F0501676F40EC1C,637603849,16
2017-09-08T09:26:19.000,-113,40EC13012680F77701F532CC2B2518,637604844,16
2017-09-07T01:00:08.000,-118,2D3FD04EA2A880EDD1D303138FBACDBAE091C2054BC064E8E9711061E6E9A8,64213485,16
2017-09-07T19:15:56.000,-119,27895EC560F4BC5C3775A1BE34CA974C37F97F972A23FFAFF1F5680E354C4AF49D0C811BCFC917C8008EADBD191C5D51048B6DF10DEA77DC66CAD7236318AD7D696889B560A428DE04C1FDC78117BB75014B2774508E5F601440937E52813F6163CE0A2377BEA4ACCA7FC5DB649DCDC7EAF2F5883BE6704153033867406A4E125948F27A2C3F154495A9D214D20B75198C6B21B4DC69E8D24145CF0D7A1636EF2D4C89A355C912C57D9E4138BFB188FF431F71FD090AAC6E30EDF74BE9CDF88B892DCEF4CC20674B1DC3EE8595C52BC3117944306783F2955F9C1841F753B3A514050B5CCDC1AE4AC380971CEE5F7DF92D7805,2708813660,16
2017-09-08T09:58:29.000,-120,400910012600900501616205DC57,637603849,16
2017-09-07T18:34:24.000,-119,40D12B012600864501AAD4704453E9FA36917A9A63263842C9A96CB535D1763FEB64,637610961,16
2017-09-08T08:03:14.000,-114,4009100126009105019E8B627680,637603849,16
2017-09-07T22:53:59.000,-118,40D12B01260087450184931B9F16E9F6B291CD2ACEEC36D17A718E5A1E5D793C2878,637610961,16
2017-09-07T20:50:25.000,-118,4009100126009205013FABF23C94,637603849,16
2017-09-09T00:37:55.000,-113,40D12B012600884501A11ACDC407B65A33C972B8084E48304C62071507BF887139CF,637610961,16
2017-09-07T20:25:30.000,-117,400910012600930501CA348CA234,637603849,16
2017-09-08T09:38:45.000,-117,516E88E34F9EF0BD1A210A75645FFD419ADFDF1315A01B2E03FE57EE88E39F9F4E761AA0EFC51553FC94FEAD58AE5027555391815001DAE5F7CEBEA681EBE0C570418AD00D3996483D918574BB2CF547AD033F41D34AE10D1C64ADC66E99624A92D6DA,1340311662,16
2017-09-07T17:17:34.000,-119,A3FB974D9786EF01C96BE828F63A3CC3AA41482E82D3A22E693F3E7D4ABF8945600E28B84BB7C370864AB445584D39597A011423D671710FC94B470985E288CB2B9A0C436531FCE517E47E95750EB99475B0D8C868B00B30934FE13A6BF0F54EB37C8B7C2F9C7E2EB77AB358DE6A8E246FE3250AEF501A9687A2EB6591C4FB26852E6F1478054E2675D1B0D25798560E9FABF12AE5A97A6A8B9DA58889A542D2FF7A8100C42E108E68F6EC32A92903E1BCC6A8AA7B8CEE448FC197ADF267C7ED3C6AF686EF5E42331BDC85865BCA95DF68992D0F88D1179A,2538444795,16
2017-09-09T08:09:48.000,-117,40D12B012600894501C0D081BF157A0B5D60C7D9E0C4A38D320EF9B5A4AD6BC67FF1,637610961,16
2017-09-06T15:07:40.000,-115,40091001260094050194AC01470A,637603849,16
2017-09-06T15:08:43.000,-119,40EC13012680F877017AC746873BE7,637604844,16
2017-09-06T15:09:45.000,-117,40091001260095050166C050A44D,637603849,16
2017-09-06T15:10:47.000,-116,40D12B0126008B4501CEF88317B68EA395F52D8A50ACA3C003C88270B314127B4E10,637610961,16
2017-09-06T15:13:54.000,-120,40091001260696050D09BF7AC93B,637603849,16
2017-09-06T15:14:56.000,-118,40D12B0126008C45016D6D3F861B87798E0569D8218B5F9399A196315169AAE28650,637610961,16
2017-09-06T15:15:59.000,-117,400910012600980501AF8632360C,637603849,16
2017-09-06T15:17:01.000,-120,40091001260099050141924773CF,637603849,16
2017-09-06T15:18:03.000,-119,4029290126801202018C0A38BBCAD2030E3CF7A57EB2E1E085,637610281,16
2017-09-06T15:19:06.000,-118,40D12B0126008E45012ADCA0249A27D865088950B08B3271752A230DEBE70B95948D,637610961,16
2017-09-06T15:20:08.000,-118,B438377FA4F9301E98C767CE2C097D9426D10DE0F58423C208698F397035D6A7283F8DF759E6D9C9519EAF645645E8F479F04B95B302A7AD6EA91976DDC00A8B170FEB9C65EDF35B4E41246822D637AE57FC245FE94E6B2044D10BC59CDE5B11CA778FB3F56A48006661718DA2BEA8EB2D78055F483409D30573AD332B000A42371411B53EB0EFABAA3E121596F4B2B07F5D61BA4DCB2147FB324CDD9FDDB1DEFA905BACDB,2759800632,16
2017-09-06T15:21:10.000,-120,408A130126A0F977327F1C8B10F00C,637604746,16
2017-09-06T15:22:12.000,-120,4009100126009A0501026E521A5F,637603849,16
2017-09-06T15:23:15.000,-120,4009100126009B05012EE877DCE2,637603849,16
2017-09-06T15:24:17.000,-115,40D12B012600904501EA08D2CD3FD4ADC0CCFB91973FBC752418EC149AEA05668464,637610961,16
2017-09-06T15:28:26.000,-121,BAAB2C3A8454F4052BF52888A5AA52F646D0BE5F713BAB42CBC8E3E9D4F34F6B4E95E8C8B3C866324B3AEE9E728FC48D1FC42E5D8D0AA5A93068985E51FA4BFC014BFE9F7E90C3AA07DFE0C18F4C6CD958,2218405035,16
2017-09-06T15:29:29.000,-115,40D12B0126009145018D4E40B1B3107D159EFFF8733CEF214F24E95A26C274CC6209,637610961,16
2017-09-06T15:31:33.000,-118,4009100126009D0501F83EE48C3E,637603849,16
2017-09-06T15:35:42.000,-120,4009100126009E0501BCA02A1301,637603849,16
2017-09-06T15:36:45.000,-117,40EC13012680FA7701B366C659BD56,637604844,16
2017-09-06T15:37:47.000,-118,4009100126009F0501B17AFECE01,637603849,16
2017-09-06T15:39:51.000,-115,40D12B012600944501CE43A9C7F59551096F18A6DB2331418DC244F4B4DDC22BEE7B,637610961,16
2017-09-06T15:40:54.000,-115,400910012600A00501F9195FFC2D,637603849,16
2017-09-06T15:41:56.000,-114,3EECD95C97BDA3596B84858E963EA862D2B27CDC6B004F8E8D32FA475107CB0A05F6EB638BDEB08CA0F197BF8B390FCAC59BE817B01914A363A6E4C4BBA39E16331334AE92FAA0B2D238B0C5A65DFFA2EAF91F4D68656018B00D20AB588795DBCC580798B17FD44D71C0ADBC345B10B1E0D8EA55699E4563E4D263228970CC03C53F23F79BDC99DAD231DA2A604A16C219194A0A690C912B311A803DF5A409A7305C54B6028816EC1C0E7C84D2C532FB54E5BB0B6C444D7AD1F1D84DC12DCC494E,2240047961,16
2017-09-06T15:42:58.000,-119,40D12B01260096450137757BF31904AB9AB03704DC144FF248B445221E316FC55E61,637610961,16
2017-09-06T15:44:01.000,-118,40D12B0126009745010E3B46C799D6B8EFEB6A908F5BCD14BF903CC16BC95B405FE8,637610961,16
2017-09-06T15:45:03.000,-120,40EC13012680FB770125F0A61F1D72,637604844,16
2017-09-06T15:46:05.000,-119,40292901268013020108E1FB4F01CCDCC42C6959104FA23DFB,637610281,16
2017-09-06T15:47:08.000,-113,400910012600A405018EAC99B335,637603849,16
2017-09-06T15:48:10.000,-114,40D12B0126009845011C228E0F8F230CFFE45B3EB2F38B3B6E12A99435B24D168FD8,637610961,16
2017-09-06T15:50:14.000,-119,AF1EA7F9F3C694A92DFBA953E8C5CD401EF05B9FC08A713ABDD169046DC6226CD4142892A480ABFD86C2851D18005E26A6457D95197DE3061FF8058A474186D6B78EB5E75F80A74A8AC35D36AB8AC81F94D994FD011B911211C3D02F04969D4980B65226A5BC4A2938087A6830937EB95A6CEAD40972BA197D8BAE04C659ED3A60160B31E4A1F2543B16A43D548A5AED151C60AEC9D839488973467A5ADFAABAF767650B32381577C13D30BD783AD473264EBFE1B9DEC68C217EE31FD1A296D55555497F3EF5BCD03BC73BE0D753E89262543B21CB059C,4093224734,16
2017-09-06T15:51:17.000,-115,400910012600A605014859BDA665,637603849,16
2017-09-06T15:54:24.000,-119,400910012600A705015DDE5275CD,637603849,16
2017-09-06T15:55:26.000,-118,40D12B0126009B8501B5FC0E73FB1D67B1CF042013280EB45D3B183AEAC1BE05711A,637610961,16
2017-09-06T15:56:28.000,-119,40EC13012680FC7701156C45177E42,637604844,16
2017-09-06T15:58:33.000,-116,400910012600A8050114B8920E2C,637603849,16
2017-09-06T15:59:35.000,-120,40C1B14E0180060008D776C57C0456596E828FE46059,21934529,16
2017-09-06T16:00:37.000,-113,400910012600A90501A460A66438,637603849,16
2017-09-06T16:01:40.000,-114,40D12B0126009D45013CC3DE7D8EBB65192464B385A3CB7225EB5D08CFCF73833725,637610961,16
2017-09-06T16:02:42.000,-117,400910012600AA05013316290201,637603849,16
2017-09-06T16:04:47.000,-119,40D12B0126009F450144306C7357FAB80632A910E5F8A21BE5442E238C6ADBF4612B,637610961,16
2017-09-06T16:05:49.000,-115,440D543516009C05015A0BDAFAE3,372593677,16
2017-09-06T16:06:51.000,-119,400910012600AD0501D04CE038A1,637603849,16
2017-09-06T16:07:54.000,-120,4029290126801402014C4767565BB1A9AA4B65D23A6241D6E2,637610281,16
2017-09-06T16:08:56.000,-119,AFD72081A36BBDADE593CC92CEBB7573E612FAE4084FC27AB2FE8E5D02BC995C709C5B4DCCB529FA9D3BE4D724D15A1A815D76789350AC5DA4292E6E5E,2743148759,16
2017-09-06T16:11:00.000,-120,48B8530F16E4811625830965FF02C8011FB07689BE2D58A446F98DEEA5E46F558EBEB85D9E05245901DFA118B1C964A3CE207F33FDF08D31DDC330C4CC9218083C9C6DD75E7A065EF8033E0A42BA3C721C94D929AC,370103224,16
2017-09-06T16:12:03.000,-118,400910012600AE0501456C14F9DA,637603849,16
2017-09-06T16:13:05.000,-115,400910012600AF05018FD6905772,637603849,16
2017-09-06T16:14:07.000,-118,40D12B012600A24501A762DD179C04699026858780F1BE0C1FE67E93033568C5546E,637610961,16
2017-09-06T16:15:10.000,-118,400910012600B0050171A59E3044,637603849,16
2017-09-06T16:16:12.000,-120,48D12B412200E34501783A6CD3183BB42C6DB969584F701123F8D7330CB7BC2A6E05,574696401,16
2017-09-06T16:17:14.000,-119,40D12B012600A445017B96F00CB2A140418F7EC87DF51F0C7A7DB3320897256C7DBF,637610961,16
2017-09-06T16:18:17.000,-117,40EC13012680FE770177544C9D6325,637604844,16
2017-09-06T16:20:21.000,-120,400910012600B205016882EB708F,637603849,16
2017-09-06T16:21:23.000,-118,40D12B012600A54501907991DD01173D23DDBD288D243050410C3247B14CC802C6C8,637610961,16
2017-09-06T16:22:26.000,-119,400910012600B30501DEA2F04FC2,637603849,16
2017-09-06T16:23:28.000,-120,40D12B012600A645018C1475A5CF37D4C0D8323A50C63D5B796F5D67BBBDC6A70D2E,637610961,16
2017-09-06T16:24:30.000,-118,400910012600B405015E7371FC26,637603849,16
2017-09-06T16:25:33.000,-118,40D12B012600A74501BF5B94D049312BF16D86CEB88B54A04B4C2143A46A71662EF6,637610961,16
2017-09-06T16:27:37.000,-118,400910012600B505017EFB0AE669,637603849,16
2017-09-06T16:29:42.000,-121,425F9B14FAC6CEC59F3D4D451FC010A1C0204C08FE5898E0A66C16644025CA7A328C06522A0EF8DD28ADE57E6DFCD5506438A9468CBC42AC8FD9F6AF668B6DF1BF891B417AFBBE5CC23AC1F189440C2FDA1B7B55A5AFF683A8FF7EF17E7F9B36B2772278C8C8DF29E5,4195654495,16
2017-09-06T16:32:49.000,-115,40D12B012600A84501CDB829BB2E1E8B4E237F9D461B891F7DACA11BF18EF92EBCBD,637610961,16
2017-09-06T16:33:51.000,-117,400910012600B605016C5FAA0D92,637603849,16
2017-09-06T16:34:53.000,-109,40EC13012680FF7701C0E25F236D3D,637604844,16
2017-09-06T16:35:56.000,-118,400910012600B70501BCCEED3A4B,637603849,16
2017-09-06T16:36:58.000,-120,402929012680150201A21D73E2294387996216EA6DC10C4023,637610281,16
2017-09-06T16:38:00.000,-118,40D12B012600AA450143B5D56674F6BC97D992312C7973595D884D544E33452BC651,637610961,16
2017-09-06T16:39:02.000,-118,400910012600B80501BC269661CF,637603849,16
2017-09-06T16:40:05.000,-119,45F918378719D448C975E2EEA9B1F6B96F50D62E82C2A687DE493F3159989760AC18A14599FFD162E879B75913D5A1F449421DAE876DCFC348A1B7588222A5843A6E8C24A304A89C0C4C803BEEE520,2268535033,16
2017-09-06T16:41:07.000,-117,400910012600B90501E741D2BD8C,637603849,16
2017-09-06T16:42:09.000,-118,40D12B012600AC4501D226BA77C2B00978C4B26835A41C39A90CD6ADD60C50D0D648,637610961,16
2017-09-06T16:43:12.000,-119,400910012600BB0501FFEB302929,637603849,16
2017-09-06T16:47:21.000,-121,40D12B012600AD4501C05BA60E7822A6B63129F2779C7B6F001AF0DE85B0E8030064,637610961,16
2017-09-06T16:48:23.000,-119,40EC130126800078014FCF49394425,637604844,16
2017-09-06T16:49:25.000,-120,400910012600BC0501251735CC2A,637603849,16
2017-09-06T16:50:28.000,-114,40D12B012600AE45014636D04AC8385E2356C9F2EBCE37B4BD8B7D9849EA888D52D2,637610961,16
2017-09-06T16:52:32.000,-119,400910012600BD050161F8D275C1,637603849,16
2017-09-06T16:53:35.000,-120,867107DA7FAF0083433FDAFC3D7BE1095F6B7303F859ECD2D653871D88721BCB001C333DD90398A08947DC9814B6B2379B7455BAC01F0DB108FAD1C35C724BC3B1AC9C1762819ED301CC7340C4C7BF57DE95A881F5B0F5A363743E77C61014DCEFF97D91DF45AD591DC32BC8968E16AAB8394808D18DF975C980F4C4FED074EB9CC0CE2557982807F36D90798965A0676087675B9662780E1C4D9A014F0EF0E8FFA676EF1E1643F6E6CC07934F9FF6ABABA1739F896739D50EF240BC328377755CFB7AB2B52C,7FDA0771,16
2017-09-06T16:54:37.000,-118,40D02B000600AF4501C2B093D07BEF0C5853FCEB6A6DC4DE997D45E365470E293258,100674512,16
