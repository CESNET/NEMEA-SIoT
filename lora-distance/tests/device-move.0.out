-119.000000,637603849,-111.000000,2017-09-06T18:39:29.000,0.050000,0,"The device 637603849 exceeded allowed distance from gateway by -2.05 dBm"
-120.000000,637604844,-113.000000,2017-09-08T09:26:19.000,0.050000,0,"The device 637604844 exceeded allowed distance from gateway by -1.00 dBm"
-119.000000,637603849,-113.000000,2017-09-06T15:47:08.000,0.050000,0,"The device 637603849 exceeded allowed distance from gateway by -0.05 dBm"
-119.000000,637603849,-113.000000,2017-09-06T16:00:37.000,0.050000,0,"The device 637603849 exceeded allowed distance from gateway by -0.05 dBm"
-120.000000,637604844,-109.000000,2017-09-06T16:34:53.000,0.050000,0,"The device 637604844 exceeded allowed distance from gateway by -5.00 dBm"
